Compilation of 2025.9.0 fails due to `maybe-uninitialized` error

### Describe the bug

When building EVerest 2025.9.0, the compilation fails as follows:
```
build/_deps/everest-core-src/lib/everest/io/src/can/socket_can_handler.cpp: In member function 'bool everest::lib::io::can::socket_can_handler::rx(everest::lib::io::can::can_dataset&)':
build/_deps/everest-core-src/lib/everest/io/src/can/socket_can_handler.cpp:81:31: error: 'can_id' may be used uninitialized [-Werror=maybe-uninitialized]
   81 |     data.set_can_id_with_flags(can_id);
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
build/_deps/everest-core-src/lib/everest/io/src/can/socket_can_handler.cpp:79:14: note: 'can_id' was declared here
   79 |     uint32_t can_id;
      |              ^~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [_deps/everest-core-build/lib/everest/io/src/CMakeFiles/everest_io.dir/build.make:132: _deps/everest-core-build/lib/everest/io/src/CMakeFiles/everest_io.dir/can/socket_can_handler.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:5159: _deps/everest-core-build/lib/everest/io/src/CMakeFiles/everest_io.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```
The issue seems to be here: https://github.com/EVerest/everest-core/blob/374030cb3e1709f436fec5fc3874a95533173f07/lib/everest/io/src/can/socket_can_handler.cpp#L78-L83

### EVerest Domain

Compilation

### Affected EVerest Module

_No response_

### To Reproduce

1. Start a build of EVerest 2025.9.0

### Anything else?

_No response_

--- a/lib/everest/io/src/can/socket_can_handler.cpp
+++ b/lib/everest/io/src/can/socket_can_handler.cpp
@@ -78,7 +78,8 @@ bool socket_can_handler::tx(can_dataset const& data) {
 bool socket_can_handler::rx(can_dataset& data) {
     uint32_t can_id = 0;
     auto status = rx(can_id, data.len8_dlc, data.payload);
-    if (status == 0) data.set_can_id_with_flags(can_id);
+    if (status == 0)
+        data.set_can_id_with_flags(can_id);
     return status == 0;
 }
 

